phpunit.xml.dist

```diff
@@ -5,6 +5,9 @@
          beStrictAboutOutputDuringTests="true"
          displayDetailsOnPhpunitDeprecations="true"
          displayDetailsOnTestsThatTriggerDeprecations="true"
+         displayDetailsOnTestsThatTriggerNotices="true"
+         displayDetailsOnTestsThatTriggerWarnings="true"
+         displayDetailsOnTestsThatTriggerErrors="true"
```

Comment:
This should be removed from this pull request, or at least this commit, it's a concern that is orthogonal to that of the main commit.